pallets/schemas/src/rpc/src/lib.rs

```diff
@@ -55,6 +55,10 @@ pub trait SchemasApi<BlockHash> {
 	/// validates a schema model and returns `true` if the model is correct.
```

Comment:
Should the response for `schemas_getBySchemaId` also include the name now? I think that will cause any issues as this is all json at this point right?

Reply:
So we don't have the access from SchemaId to SchemaName. We only have the mapping from name to ids.